metaflow/package.py

```diff
@@ -75,6 +89,27 @@ def path_tuples(self):
                 addl_suffixes=self.metaflow_extensions_addl_suffixes,
             ):
                 yield path_tuple
+
+        # Any custom packages exposed via decorators
+        deco_module_paths = {}
+        for step in self._flow:
+            for deco in step.decorators:
+                for path_tuple in deco.add_to_package():
+                    file_path, file_name = path_tuple
+                    # Check if the path is not duplicated as
+                    # many steps can have the same packages being imported
+                    if file_name not in deco_module_paths:
+                        deco_module_paths[file_name] = file_path
+                        yield path_tuple
+                    elif deco_module_paths[file_name] == file_path:
+                        pass
+                    elif (
```

Comment:
should replace just with "else:"

Reply:
+1

Reply:
With a just `else` we will throw an Exception even if `deco_module_paths[file_name] == file_path` (which we don't want to) . Rather I can change it to `elif deco_module_paths[file_name] != file_path`; this will keep logic the same